MekHQ/src/mekhq/campaign/autoresolve/acar/report/MoraleReporter.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2024 - The MegaMek Team. All Rights Reserved.
+ *
+ *  This file is part of MekHQ.
+ *
+ *  MekHQ is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  MekHQ is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with MekHQ. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package mekhq.campaign.autoresolve.acar.report;
+
+import megamek.common.IGame;
+import megamek.common.Roll;
+import mekhq.campaign.autoresolve.component.Formation;
+
+import java.util.function.Consumer;
+
+public class MoraleReporter {
+
+    private final IGame game;
+    private final Consumer<PublicReportEntry> reportConsumer;
+
+    public MoraleReporter(IGame game, Consumer<PublicReportEntry> reportConsumer) {
+        this.game = game;
+        this.reportConsumer = reportConsumer;
+    }
+
+    public void reportMoraleCheckStart(Formation formation, int toHitValue) {
+        // 4500: Start of morale check
```

Comment:
What are these integers in reference to? This is the type of thing that probably makes absolute sense to you, as the writer, but means nothing to anyone just looking at your comment :D

Reply:
<img width="1002" alt="Screenshot 2024-12-16 at 19 56 27" src="https://github.com/user-attachments/assets/a08dc46c-8085-49c5-ad77-87e7778ec818" />

I was using those to guide me, so I could go back in the code and note all the message numbers I had to write them down in the properties file when I finished.